content/2_Bronze/Intro_Sets.mdx

```diff
@@ -32,80 +52,125 @@ A **set** is a collection of elements that contains no duplicates.
 
 <CPPSection>
 
-In _sorted_ sets, the elements are sorted in order of element. Insertions,
-deletions, and searches are all $\mathcal{O}(\log N)$, where $N$ is the number
-of elements in the set.
+### Unordered Sets
+
+In _unordered_ sets, elements are stored in a random order. Insertions,
+deletions, and searches are all $\mathcal{O}(1)$. Unordered sets are implemented
+by
+[`std:unordered_set`](https://www.cplusplus.com/reference/unordered_set/unordered_set/)
+in the `<unordered_set>` header.
 
-[`std::set`](http://www.cplusplus.com/reference/set/set/) implements sorted
-sets in C++. Some operations on an `std::set` named `s` include:
+Some operations on an `std::unordered_set` named `s` include:
 
 - `s.insert(x)`, which adds the element `x` to `s` if not already present.
 - `s.erase(x)`, which removes the element `x` from `s` if present.
 - `s.count(x)`, which returns `1` if `s` contains `x` and `0` if it doesn't.
 
-You can iterate through a set in sorted order using a for-each loop.
-
 ```cpp
-set<int> s;
-s.insert(1);  // [1]
-s.insert(4);  // [1, 4]
-s.insert(2);  // [1, 2, 4]
-s.insert(1);  // [1, 2, 4]
+unordered_set<int> s;
+s.insert(1);  // {1}
+s.insert(4);  // {1, 4}
+s.insert(2);  // {1, 4, 2}
+s.insert(1);  // {1, 4, 2}
 // the insert method did nothing because 1 was already in the set
 cout << s.count(1) << endl;  // 1
 s.erase(1);  // [2, 4]
 cout << s.count(5) << endl;  // 0
 s.erase(0);  // [2, 4]
 // if the element to be erased was not present, nothing happens
+```
+
+### Sorted Sets
+
+In _sorted_ sets, the elements are sorted in order of element. Insertions,
+deletions, and searches are all $\mathcal{O}(\log N)$, where $N$ is the number
+of elements in the set. Sorted sets are implemented by
+[`std:set`](https://www.cplusplus.com/reference/set/set/) in the `<set>`
+header.
+
+`std::set` includes all the essential operations that `std::unordered_set` has
+(including insertion, deletion, searches, etc.), but also some additional ones.
+Refer to the
+[More Operations on Sorted Sets](https://usaco.guide/silver/intro-sorted-sets)
+module for more detail.
+
+You can iterate through a set in sorted order using a for-each loop.
+
+```cpp
+set<int> s;
+s.insert(1);  // [1]
+s.insert(4);  // [1, 4]
+s.insert(2);  // [1, 2, 4]
```

Comment:
```suggestion
s.insert(1);  // {1}
s.insert(4);  // {1, 4}
s.insert(2);  // {1, 2, 4}

Reply:
actually imma revert this since i think the point of square brackets is to emphasize that the set is ordered (this is the same everywhere else)